### DISCUSSION TITLE
Welcome to SCPP Discussions!

### DISCUSSION THREAD

**Discussion by Its-Jakey** [Its-Jakey/SCPP-old-#1] [General]
Just a discussion tab for questions/comments/anything

**Comment by MinekPo1** [comment]
Just a small thing that does not deserve its own thread but what is called a parser in the code is actually an assembler and the assembler just formats the assembled vm instructions.

**Reply by MinekPo1** [reply]
My mind is playing tricks on me, there is nothing called a parser haha! However parser behaviour of antlr are used to create assembler functionality.

**Reply by MinekPo1** [reply]
nvm my original comment was correct
